src/components/CommentList/CommentList.jsx

```diff
@@ -1 +1,18 @@
-export const CommentList = () => <>Put the list here</>;
+import { CommentInfo } from '../CommentInfo/CommentInfo';
+import './CommentList.scss';
+
+export const CommentList = ({ IdOfPost, Comments }) => {
+  const comms = Comments.filter(com => com.postId === IdOfPost);
```

Comment:
```suggestion
  const comms = comments.filter(com => com.postId === idOfPost);
```
name props in camelCase